src/odemis/acq/stream/_sync.py

```diff
@@ -167,6 +167,7 @@ def __init__(self, name, streams):
         self._acq_done = threading.Event()
 
         # Start threading event for live update overlay
+        self._current_scan_area = (0, 0, 0, 0)  # currently scanned area location based on px_idx
```

Comment:
Furthermore why not define _current_scan_area not as (None,None,None,None)? Since at init no area is defined? (just out of curriosity I don't think this is essential to implement)

Reply:
I've changed it to `None` by default. I don't think it's really useful to write `(None,None,None,None)`. Either it contains something invalid (`None`) or it contains a tuple with ints. No need to be half-way.